primitives/src/v4/metrics.rs

```diff
@@ -176,4 +188,14 @@ pub mod metric_definitions {
 			description: "Counts the number of bitfields signature checked in `enter_inner`.",
 			labels: &["validity"],
 		};
+
+	/// Measures how much time does it take to verify a single validator signature of a dispute statement
+	pub const PARACHAIN_VERIFY_DISPUTE_SIGNATURE: HistogramDefinition =
+		HistogramDefinition {
+			name: "polkadot_parachain_verify_dispute_signature",
+			description: "How much time does it take to verify a single validator signature of a dispute statement, in microseconds",
+			buckets: &[10.0, 50.0, 100.0, 200.0, 400.0, 800.0, 1000.0, 1500.0, 2000.0, 2500.0],
```

Comment:
This is the distribution from a zombienet run on my machine. We can readjust after testing on Versi
```
# TYPE polkadot_parachain_verify_dispute_signature histogram
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="10"} 0
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="50"} 0
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="100"} 59
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="200"} 162
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="400"} 165
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="800"} 166
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="1000"} 167
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="1500"} 167
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="2000"} 167
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="2500"} 167
polkadot_parachain_verify_dispute_signature_bucket{chain="rococo_local_testnet",le="+Inf"} 167
polkadot_parachain_verify_dispute_signature_sum{chain="rococo_local_testnet"} 19951
polkadot_parachain_verify_dispute_signature_count{chain="rococo_local_testnet"} 167
100  534k  100  534k    0     0   118M      0 --:--:-- --:--:-- --:--:--  130M
```

Reply:
On second thought - it will be worse on versi...
Will it be better to use mircoseconds as dimensions and have buckets like `[1, 5, 10, 50, 100, 200, 300, 400, 500, 600, 700, 800, 1000]`?

Reply:
Remark by @sandreim -> these should look like a time histogram so the dimension should match with it (either secs or ms).